platform_versioned_docs/version-24.1.0/enterprise/data-studios.mdx

```diff
@@ -90,16 +109,32 @@ This procedure describes how to configure Data Studios for Kubernetes deployment
       INITIAL_ACCESS_TOKEN: <INITIAL_ACCESS_TOKEN>
     ```
 
-1. Edit your ConfigMap named `platform-backend-cfg` for Platform and add the following environment variables:
+1. Edit the ConfigMap named `platform-backend-cfg` in the `configmap.yml` for Platform.
+    -  Add the following environment variables:
+
+       - `TOWER_DATA_STUDIO_CONNECT_URL`
+       - `TOWER_OIDC_REGISTRATION_INITIAL_ACCESS_TOKEN`: Use the same access token that you specified in the previous step
+       - `TOWER_OIDC_PEM_PATH`
 
-  ```yaml
-    TOWER_DATA_STUDIO_ALLOWED_WORKSPACES=<ALLOWED WORKSPACES>
-    TOWER_DATA_STUDIO_CONNECT_URL=<YOUR CONNECT URL, SAME AS `CONNECT_PROXY_URL` IN `data-studios.env`>
-    TOWER_OIDC_REGISTRATION_INITIAL_ACCESS_TOKEN=<YOUR INITIAL ACCESS TOKEN, IDENTICAL AS `INITIAL_ACCESS_TOKEN` in `data-studios.env`>
-    TOWER_OIDC_PEM_PATH=<PATH TO PEM FILE>
-  ```
+    - If you want to restrict Data Studios to specific organization workspaces, list each workspace ID in the `TOWER_DATA_STUDIO_ALLOWED_WORKSPACES`.
+    - If you want to allow access to Data Studios in every organizational workspace, do not include the `TOWER_DATA_STUDIO_ALLOWED_WORKSPACES`.
 
-1. Apply your updated configuration: `kubectl apply -f configmap.yml`
+1. Optional: If you want to allow access to Data Studios in every organizational workspace, edit the ConfigMap named `tower-yml` in the `configmap.yml` and include the following stanza with your existing configuration:
```

Comment:
Guidance on why they should choose one option over the other re: TOWER_DATA_STUDIO_ALLOWED_WORKSPACES?

Reply:
The platform interpret the value of the configuration as:
- `null`: all enabled
- empty list: all disabled
- list of values: enable the listed values

Due to the limitations of environment variables the first case `null` cannot be specified, so the yaml file is needed, users are not forced to use env variables for the other two cases:

All disabled:
```
tower:
  data-studio:
     allowed-workspaces: []
```

Enable only some workspaces:
```
tower:
  data-studio:
     allowed-workspaces: [ 12321, 231231]
```

The advantage of the env variable is that is shorter, but maybe the two alternatives ways that we are forced to use are confusing.

Reply:
@t0randr, having several approaches where only one way covers all cases is a bit confusing.

Reply:
@justinegeffen @llewellyn-sl 

I'm strongly tempted to just omit the environment variable for this entirely as it provides only a subset of the functionality, and it makes the procedure complex and bifurcated. Customers are probably already using `tower.yml` anyway for site-specific configuration, so it won't be a new thing.